audformat/core/database.py

```diff
@@ -734,14 +739,70 @@ def join_field(
             assert_equal(other, 'license')
             assert_equal(other, 'usage')
 
-        # can only join databases with relatvie paths
+        # can only join databases with relative paths
         for database in [self] + others:
             if not database.is_portable:
                 raise RuntimeError(
                     f"You can only update with databases that are portable. "
                     f"The database '{database.name}' is not portable."
                 )
 
+        # join schemes with labels
+        for other in others:
+            for scheme_id in other.schemes:
+                if scheme_id in self.schemes:
+                    other_labels = other.schemes[scheme_id].labels
+                    self_labels = self.schemes[scheme_id].labels
+
+                    # join labels from misc tables
+                    if (
+                        isinstance(other_labels, str)
+                        or isinstance(self_labels, str)
+                    ):
+
+                        if (
+                            isinstance(other_labels, str)
+                            != isinstance(self_labels, str)
+                        ):
+                            raise ValueError(
+                                f"Cannot join scheme "
+                                f"'{scheme_id}' "
+                                f"when one is using a misc table "
+                                f"and the other is not."
+                            )
+
+                        if other_labels != self_labels:
+                            raise ValueError(
+                                f"Cannot join scheme "
+                                f"'{scheme_id}' "
+                                f"when using misc tables "
+                                f"with different IDs: "
+                                f"'{self_labels}' "
+                                f"!= "
+                                f"'{other_labels}'."
+                            )
+
+                        other_table = other[other_labels]
+                        self_table = self[self_labels]
+                        index = utils.union(
+                            [
+                                other_table.index,
+                                self_table.index,
+                            ],
+                        )
+                        other_table.extend_index(index, inplace=True)
+                        self_table.extend_index(index, inplace=True)
+                        # ensure same index order in both tables
+                        other_table._df = other_table.df.reindex(index)
+                        self_table._df = self_table.df.reindex(index)
```

Comment:
You said it cannot happen when updating a database that the columns of the tables are different, so we don't have to care here.
I guess the reason is that it will raise an error when updating the misc tables.
If this is the case, could you maybe add a comment to the code here that we only need to update the index of the table as the columns are handled somewhere else.

Reply:
Correct, at this stage we only ensure that the indices are combined. This is required later on when we update the tables as they assume that the dtypes of the indices are alike. I added the following comment:

```
                    # join labels from misc tables
                    # by combining the index of all tables,
                    # columns values will be updated
                    # later when the tables are joined
```